cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/keyspaces/CradleInfoKeyspaceCreator.java

```diff
@@ -46,15 +46,12 @@ protected void createTables() throws IOException
 	@Override
 	public void createAll() throws IOException, CradleStorageException
 	{
-		try
-		{
-			createKeyspace();
-			createTables();
-		}
-		catch (CradleStorageException e)
+		if (getKeyspaceMetadata() != null)
 		{
 			logger.info("Existing \"Cradle Info\" keyspace '{}' is in use", getKeyspace());
```

Comment:
I'm in doubt about the text in log.
`\"Cradle Info\" keyspace '{}' already exists`, may be?